### PR TITLE
Final edits before release

### PR DESCRIPTION
**Pull request by stitam**
PR task list:
- [x] Update NEWS
- [x] Add tests (if appropriate)
- [x] Update documentation with `devtools::document()`
- [x] Check package passed

### CODE REVIEW

**Comment by stitam** [comment]
I don't know how to mark the PR as draft, but this is a draft. This will be the last PR to merge before the release.

File: R/alanwood.R [hunk 1]
```diff
@@ -16,6 +16,10 @@
 #' @note for type = 'cas' only the first matched link is returned.
 #' Please respect Copyright, Terms and Conditions
 #' \url{http://www.alanwood.net/pesticides/legal.html}!
+#' @references Eduard Szocs, Tamas Stirling, Eric R. Scott, Andreas Scharmuller,
```
Comment (andschar) [inline R/alanwood.R:19]: there is an __ü__ instead of an __u__ in my last name: Scharmüller

Comment (andschar) [reply R/alanwood.R:19]: Or are the Umlauts (ä, ü,..) not allowed in \@references?

**Comment by stitam** [comment]
> Does this PR also update the output of `citation("webchem")`?

Yes, inst/CITATION is included in the PR.

File: R/alanwood.R (hunk 1 above)
Comment (stitam) [reply R/alanwood.R:19]: And I am Tamás not Tamas:D. I followed Eduard's example and didn't add any special characters, but I'll check and correct if they are allowed.

Comment (stitam) [reply R/alanwood.R]: It didn't throw any errors so far, so done I guess:)

**Pull request merged**